cmake/IntegratedOpenCL.cmake

```diff
@@ -126,6 +128,7 @@ list(
     "libs/lexical_cast"
     "libs/math"
     "libs/move"
+    "libs/mp11"
```

Comment:
Starting from `1.81` `container_hash` module requires `describe` and `mp11` modules
https://github.com/boostorg/container_hash/blob/bd5b7a359c285a65836674d963e1137b067983ac/CMakeLists.txt#L18-L19